src/gurt/tests/test_gurt.c

```diff
@@ -31,7 +31,7 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <pthread.h>
-#include "wrap_cmocka.h"
+#include "../../cart/src/utest/wrap_cmocka.h"
```

Comment:
hmm...just add CPPPATH in cart SConscript to global env or move the file here ?

Reply:
Added to CPPPATH but in src/SConscript to global env instead because gurt is compiled first, and is used also in cart